src/components/Board.tsx

```diff
@@ -0,0 +1,136 @@
+import React, { useEffect, useState } from 'react'
+import Square from './Square'
+import { checkWinner, checkStale } from '../utils'
```

Comment:
if you add `"baseUrl": "src",` in your tsconfig inside `compilerOptions` you can do absolute imports  from src like:
```suggestion
import { checkWinner, checkStale } from 'utils'
```